Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = [
   "bin/node",
   "bin/faucet",
   "crates/block-producer",
+  "crates/miden-rpc-proto",
```

Comment:
nit: since we don't prefix other crates with `miden-`, I'm not sure we need to do it here either. We should probably change this to `rpc-proto` (the name of the crate would still be `miden-rpc-proto`).